processes/gops/process_info_linux.go

```diff
@@ -0,0 +1,20 @@
+package gops
+
+import (
+	"strings"
+
+	"github.com/shirou/gopsutil/process"
+)
+
+func pickName(p *process.Process) (string, error) {
+	cmdline, err := p.Cmdline()
+	if err != nil {
+		return "", err
+	}
+	// Assume that the process is a kernel process when it has no args
+	if strings.TrimSpace(cmdline) == "" {
```

Comment:
Actually `gopsutil` doesn't wrap kernel process names with square brackets, so I don't really know how to ID kernel processes in a more reliable way...

Also, it seems that the `ps` command wraps the name in square brackets precisely when the process has no args. Found this in the man page of the Linux `ps` command, in the description of the `COMMAND` field:

> Sometimes the process args will be unavailable; when this happens, ps will instead print the executable name in brackets.

That's actually the reason why I'm using this criteria, but it's certainly not perfect as a non-kernel process could also be running without args.